debian/changelog

```diff
@@ -1,3 +1,51 @@
+yaru-theme (19.10.2) UNRELEASED; urgency=medium
+
+  [ Feichtmeier ]
+  * Provide yaru checkbox assets for the shell
+  * Provide yaru checkboxes for the dark shell
+  * Use correct paths + improve SVGs
+  * Light hb in light theme
+  * Fix asset paths and blue shift keys
+  * Green enter key, white enter asset
+  * Adapt gtk2 menu colors
+  * Use @madsrh headerbar bg colors and border
+  * OSDkeyboard asset changes
+  * Hover/Active color changes
+  * Change shell active gray
+  * New @madsrh hb bg color
+  * Use fg_color for hovered menuitems
+  * Re-add adwaita custom code for search entries
+  * Sync bg_color with gtk popover colors
+  * OSD color and app notification changes
+  * Color changes
```

Comment:
up to you, but maybe consider deleting entries like this which you can't understand just by reading the debian/changelog